klippy/serialhdl.py

```diff
@@ -34,6 +34,12 @@ def __init__(self, reactor, warn_prefix=""):
         self.last_notify_id = 0
         self.pending_notifications = {}
     def _bg_thread(self):
+        try:
+            wp = self.warn_prefix.replace("'",'').replace(":",'')
```

Comment:
perhaps a comment here with an example about the type of string you're trying to manipulate here would be nice

Reply:
Mmm,
```
        wp = "mcu '%s': " % (self._name)
        self._serial = serialhdl.SerialReader(self._reactor, warn_prefix=wp)
```
But I definitely could add it.